indico/modules/rb_new/client/js/common/bookings/BookingEditForm.jsx

```diff
@@ -207,7 +207,6 @@ class BookingEditForm extends React.Component {
         const bookedByCurrentUser = sessionUser.id === bookedForUser.id;
         const today = moment();
         const bookingFinished = today.isAfter(startDt, 'day') && (endDt && today.isAfter(endDt, 'day'));
-
```

Comment:
is this one intentional? ;)

Reply:
I never intend to change already existing spacing in parts of the code. I should be more careful when adding to git. Thanks!

Reply:
`git add -p` ;)

Reply:
I know but sometimes when there's a bulk of code along with it you focus on the important parts and miss those little line adds/deletes. That's what I meant by more careful.

Reply:
@panagiotappl I usually browse through my changes once more just before creating the PR. It's easy to spot unintended changes then.

Reply:
That's a good tip @meluru